Formula/rabbitmq-c.rb

```diff
@@ -4,6 +4,7 @@ class RabbitmqC < Formula
   url "https://github.com/alanxz/rabbitmq-c/archive/v0.10.0.tar.gz"
   sha256 "6455efbaebad8891c59f274a852b75b5cc51f4d669dfc78d2ae7e6cc97fcd8c0"
   license "MIT"
+  revision 1
```

Comment:
```suggestion
```
We can drop the revision, as there's no need to dispose of the old bottles here.

Reply:
dropped revision